ui/partials/main/thPinboardPanel.html

```diff
@@ -66,23 +66,30 @@
      title="{{!hasPinnedJobs() ? 'No jobs pinned' : ''}}">
   <div class="btn-group">
     <button class="btn btn-default btn-xs save-btn"
-          title="{{ !hasPinnedJobs() ? 'No job selected' : 'Save all classification data' }}"
-          ng-click="save()"
-          ng-disabled="!canSaveClassifications()">save
+            title="{{ !user.loggedin && !canSaveClassifications() ?
+                      'Not logged in / default classification data / no job pinned' : 'Save all classification data' }}"
+            ng-click="save()"
+            ng-disabled=" !user.loggedin && !canSaveClassifications()">save
     </button>
     <button class="btn btn-default btn-xs dropdown-toggle save-btn-dropdown"
-          title="{{ !hasPinnedJobs() ? 'No pinned jobs' : 'Additional pinboard functions' }}"
-          ng-disabled="!hasPinnedJobs()"
-          data-toggle="dropdown">
+            title="{{ !hasPinnedJobs() ? 'No pinned jobs' : 'Additional pinboard functions' }}"
+            ng-disabled="!hasPinnedJobs()"
+            data-toggle="dropdown">
       <span class="caret"></span>
     </button>
     <ul class="dropdown-menu pull-right" role="menu">
-      <li><a ng-click="saveClassificationOnly()">Save classification only</a></li>
-      <li class="{{ !hasRelatedBugs() ? 'disabled' : '' }}"
-          title="{{ !hasRelatedBugs() ? 'No bugs selected' : '' }}">
-          <a ng-click="hasRelatedBugs() && saveBugsOnly()">Save bugs only</a>
-      </li>
-      <li><a ng-click="retriggerAllPinnedJobs()">Retrigger all</a></li>
+      <li class="{{ !user.loggedin && !canSaveClassifications() ? 'disabled' : '' }}"
+          title="{{ !user.loggedin && !canSaveClassifications() ?
+                    'Not logged in / default classification data / no job pinned' : 'Save classification only' }}">
+        <a ng-click="!canSaveClassifications() || saveClassificationOnly()">Save classification only</a></li>
+      <li class="{{ !user.loggedin || !hasRelatedBugs() ? 'disabled' : '' }}"
+          title="{{ !user.loggedin || !hasRelatedBugs() ?
+                    'Not logged in / no related bugs added' : 'Save bug classification data' }}">
+        <a ng-click="!canSaveClassifications() || saveBugsOnly()">Save bugs only</a></li>
+      <li class="{{ !user.loggedin ? 'disabled' : '' }}"
+          title="{{ !user.loggedin ? 'Must be logged in to retrigger all pinned jobs' : 'Repeat the pinned jobs'}}">
```

Comment:
I would just say "Not logged in" instead of "Must be logged in to..."

No point in being overly verbose here.